cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -291,6 +291,9 @@ class HydratorPlusPlusConfigStore {
         config.checkpointDir = this.getCheckpointDir();
       }
       config.stopGracefully = this.getGracefulStop();
+    } else if (appType === this.GLOBALS.eltSqlPipeline) {
```

Comment:
I don't think this block will ever get executed since the first condition on line 255 is checking for batch, and this is considered batch, right?

Reply:
Not applicable as sql-pipeline is not a batch pipeline.